cmd/collect/work.go

```diff
@@ -2,24 +2,111 @@ package main
 
 import (
 	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
 
 	"github.com/GSA-TTS/jemison/internal/common"
 	"github.com/riverqueue/river"
+	"github.com/xeipuuv/gojsonschema"
 	"go.uber.org/zap"
 )
 
 //nolint:revive
 func (w *CollectWorker) Work(ctx context.Context, job *river.Job[common.CollectArgs]) error {
-	zap.L().Info("Collect has access to crawl record",
-		zap.String("scheme", job.Args.Scheme),
-		zap.String("host", job.Args.Host),
-		zap.String("path", job.Args.Path))
+	zap.L().Info("Collect Worker processing job", zap.String("data-json", job.Args.JSON))
 
-	if err := HandleBusinessLogic(job.Args); err != nil {
-		zap.L().Error("failed to handle business logic", zap.Error(err))
+	// Ensure schemas are initialized
+	if err := ensureSchemasInitialized(); err != nil {
+		return err
+	}
+
+	// Deserialize JSON data
+	rawData, err := deserializeJSON(job.Args.JSON)
```

Comment:
Does this field have multiple names/representations? E.g. `RawData` in one place and `JSON` in another? It might be worth aligning them. My naming in the past hasn't been as consistent as I'd like, so maybe do a quick check and make sure we're not introducing another place where we have multiple names for the same thing?